ui/app/AppLayouts/Chat/components/PrivateChatPopupExistingContacts.qml

```diff
@@ -0,0 +1,63 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.13
+import "../../../../imports"
+import "../../../../shared"
+import "../../../../shared/status"
+import "./"
+
+Rectangle {
+    id: root
+    anchors.left: parent.left
+    anchors.right: parent.right
+    property string filterText: ""
+    property bool expanded: true
+    signal contactClicked(var contact)
+
+    function matchesAlias(name, filter) {
+        let parts = name.split(" ")
+        return parts.some(p => p.startsWith(filter))
+    }
+
+    height: Math.min(contactListView.contentHeight, (expanded ? 320 : 192))
+    ScrollView {
+        anchors.fill: parent
+        Layout.fillWidth: true
+        Layout.fillHeight: true
```

Comment:
The Layout stuff is useless if you are not in a `LayoutSomething` component. Anyway, you have `anchors.fill` that does it already.

Reply:
This might be a left over when I've moved stuff around. Will update